src/Redistribution.sol

```diff
@@ -622,14 +693,14 @@ contract Redistribution is AccessControl, Pausable {
      * @notice The number of the current round.
      */
     function currentRound() public view returns (uint256) {
```

Comment:
you changed all round related variables to `uint32` from `uint256`. why did left this out?

Reply:
Generally, I changed the storage variables so we save gas, these are view and/or memory ones that don't change much for the optimization of gas if I change it to a lower unit.

But the main thing is that cr and currentRound() are used in many places so I would need to downcast a lot of other variables and how other functions work or upcast it again from 32 to 256 to keep them how they work now. Was afraid I could introduce some errors that way. 
It could be done, just need to be careful not to mess up something and I didn't get to that so far and take it as low priority.